src/main/java/com/elixr/poc/util/PropertiesUtil.java

```diff
@@ -0,0 +1,39 @@
+package com.elixr.poc.util;
+
+import lombok.extern.log4j.Log4j2;
+import org.apache.catalina.core.Constants;
+
+import java.util.Properties;
+
+@Log4j2
```

Comment:
Why is this annotation? Why not slf4j? What is the difference?

Reply:
Solved